tests/moses/main/ASMOSESUTest.cxxtest

```diff
@@ -99,7 +99,7 @@ public:
 	}
 	void test_atomese_sr_3() {
 		cout << BOOST_CURRENT_FUNCTION << endl;
-		moses_test_score({"-Hsr", "-k3", "-m100000", "-L1", "--atomspace-port=1"});
+		moses_test_score({"-Hsr", "-k3", "-m100000", "-L1", "--atomspace-port=1", "--atomspace-store=1"});
```

Comment:
I think `--atomspace-store` should be enabled by default, without having to ask the user to enable it, well because I suspect it is gonna be more efficient that way (I hope I'm right!).